tests/sentry/sentry_apps/tasks/test_sentry_apps.py

```diff
@@ -760,7 +760,8 @@ def test_saves_error_if_webhook_request_with_json_content_fails(self, safe_urlop
         assert first_request["response_code"] == 400
         assert first_request["event_type"] == "issue.assigned"
         assert first_request["organization_id"] == self.install.organization_id
-        assert json.loads(first_request["response_body"]) == json_content
+        if first_request["response_body"]:
+            assert json.loads(first_request["response_body"]) == json_content
```

Comment:
q: why did this change ?

Reply:
since `response_body` is NotRequired we have to check before passing it into `json.loads` 🫠  mypy things

Reply:
if `response_body` is supposed to exist for this test case, could we assert `response_body` instead? And then do the loaders? Like there could potentially be an error that falls through the cracks if we do if statements and then asserts.

Reply:
that's true i'll fix this